modules/extension/xsd/xsd-gml2/src/test/java/org/geotools/gml2/ApplicationSchemaXSDTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2002-2008, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.gml2;
+
+import static org.junit.Assert.*;
+
+import org.geotools.xsd.SchemaLocationResolver;
+import org.junit.Test;
+
+public class ApplicationSchemaXSDTest {
+
+    @Test
```

Comment:
Please add a test describing what this class is testing.

Reply:
What do you mean? A comment?

Reply:
Yes a comment, thx.